helix-term/src/ui/picker.rs

```diff
@@ -124,6 +124,7 @@ impl<T: 'static> Component for FilePicker<T> {
         }) {
             // align to middle
             let first_line = line
```

Comment:
This looks odd and indicates that the lines passed in the result were likely wrongly calculated

Reply:
Here's the problem this addresses:
1. DAP gives us stack pointer located in a Rust library code (`/rustc/...`, obviously not found)
2. Picker tries to center the content on line 148, while doc is empty
3. Crash!